tsql/TSqlLexer.g4

```diff
@@ -808,7 +816,7 @@ XSINIL:                                'XSINIL';
 DOLLAR_ACTION:                         '$ACTION';
 
 SPACE:              [ \t\r\n]+    -> skip;
-COMMENT:            '/*' .*? '*/' -> channel(HIDDEN);
+COMMENT:            '/*' (COMMENT|.)*? '*/' -> channel(HIDDEN);
```

Comment:
Could you add a link to nested comment description? https://docs.microsoft.com/en-us/sql/t-sql/language-elements/slash-star-comment-transact-sql
Spaces before and after vertical bar: `COMMENT | .`.

Reply:
Done.